server/src/prisma/migrations/20240306001420_delete_expense_payments/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
I think https://github.com/HAK2024/fairy-share/pull/59 changes also include in this PR. If you change the heading branch from `main` to `feature/change-expensePayment-schema`, it should be solved. Please try not to have unnecessary or duplicated changes in other PR from the next time 🙏

Reply:
Oh I was wondering how to not make dupulicated changes!
You mean I should make a branch from feature/change-expensePayment-schema?

Reply:
You already made this branch from main branch so you can not make a branch from `feature/change-expensePayment-schema`. Instead, you can merge `feature/change-expensePayment-schema` to this branch and change the heading branch to `feature/change-expensePayment-schema`. Now you want to merge to main so there are changes here. Hope makes sense...!